ui/admin/app/components/form/onboarding/index.hbs

```diff
@@ -0,0 +1,60 @@
+<Rose::Form
+  class='full-width'
+  @cancel={{@cancel}}
+  @disabled={{@model.target.isSaving}}
```

Comment:
Do we have a model? I didn't see one being passed into the form

Reply:
Good catch, my bad.

Changed to:
- Passing model.
- Not using `@disabled` in this form since it does not apply. Or at least not with the current implementation, correct me if I am wrong.